app/src/main/java/com/amaze/filemanager/ui/views/drawer/Drawer.java

```diff
@@ -521,6 +523,50 @@ public void refreshDrawer() {
         R.drawable.ic_ftp_white_24dp,
         null);
 
+    addNewItem(
+            menu,
+            LASTGROUP,
+            order++,
+            R.string.wifip2p,
+            new MenuMetadata(
+                    () -> {
+                      boolean isAUInstalled = Utils.appInstalledOrNot(AboutActivity.PACKAGE_AMAZE_UTILS,
+                              mainActivity.getPackageManager());
+                      if (isAUInstalled) {
+                        try {
+                          Utils.openURL("amaze://teamamaze.xyz/transfer", mainActivity);
+                        } catch (ActivityNotFoundException e) {
+                          GeneralDialogCreation.showDownloadAmazeUtilsDialog(mainActivity);
+                        }
+                      } else {
+                        GeneralDialogCreation.showDownloadAmazeUtilsDialog(mainActivity);
+                      }
+                    }),
+            R.drawable.ic_round_connect_without_contact_24,
+            null);
+
+    addNewItem(
+            menu,
+            LASTGROUP,
+            order++,
+            R.string.analyse_storage,
+            new MenuMetadata(
+                    () -> {
+                      boolean isAUInstalled = Utils.appInstalledOrNot(AboutActivity.PACKAGE_AMAZE_UTILS,
+                              mainActivity.getPackageManager());
+                      if (isAUInstalled) {
+                        try {
+                          Utils.openURL("amaze://teamamaze.xyz/analyse", mainActivity);
+                        } catch (ActivityNotFoundException e) {
+                          GeneralDialogCreation.showDownloadAmazeUtilsDialog(mainActivity);
```

Comment:
When does this Exception happen? Shouldn't `isAUInstalled` prevent this?

Reply:
yes. this happens in case the deeplink can't be handled by amaze utils, but it is installed.
For people using old version of AU.